lxd/api_cluster.go

```diff
@@ -4237,3 +4247,67 @@ func clusterGroupValidateName(name string) error {
 
 	return nil
 }
+
+func clusterSelectTarget(ctx context.Context, s *state.State, gateway *cluster.Gateway, inst instance.Instance, candidateMembers []db.NodeInfo) (*db.NodeInfo, error) {
```

Comment:
Can you call this `evacuateClusterSelectTarget` so its clear its to do with evacuation and shares the same prefix with `evacuateClusterMember`.